presto-main/src/main/java/com/facebook/presto/execution/scheduler/SourcePartitionedScheduler.java

```diff
@@ -437,14 +437,15 @@ public synchronized void notifyAllLifespansFinishedExecution()
                 .addAll(noMoreSplitsNotification.keySet())
                 .build();
         for (InternalNode node : nodes) {
-            // source partitioned tasks can only receive broadcast data; otherwise it would have a different distribution
             ImmutableMultimap<PlanNodeId, Split> splits = ImmutableMultimap.<PlanNodeId, Split>builder()
                     .putAll(partitionedNode, splitAssignment.get(node))
                     .build();
+
```

Comment:
Yet these seem to be unrelated changes ;)

Reply:
Yeah. I found without empty lines it's hard to read sometimes...Squash them into the first commit (it's not real "refactor" but i don't want to make commit title too verbose, like "minor refactor and style adjustment...")  and mention it in the commit message :)